src/v2/layouts/MemberDataGrid/MembersActionWrapper/MemberStatusModal/StatusModal.jsx

```diff
@@ -0,0 +1,56 @@
+import PropTypes from 'prop-types';
+import { useContext } from 'react';
+import { Form, Formik } from 'formik';
+import { CustomInput } from '../../../../components/design/form/CustomInputs';
+import { StringsContext } from '../../../../components/providers/StringsProvider/StringsContext';
+
+export const StatusModal = ({ getSelectedStatus, status }) => {
```

Comment:
Change the directory name as well. Also, follow the export from index.js pattern
```suggestion
export const MemberStatusRadioDropdown = ({ getSelectedStatus, status }) => {
```